src/app/components/pages/TeacherRegistration.tsx

```diff
@@ -216,18 +221,22 @@ export const TeacherRegistrationBody = () => {
 
   const handleSubmit = (event: React.FormEvent<HTMLFormElement>) => {
     event.preventDefault();
-    register({
-      registrationUser: registrationUser,
-      unverifiedPassword: unverifiedPassword,
-      userContexts: userContexts,
-      dobOver13CheckboxChecked: dobOver13CheckboxChecked,
-      emailPreferences: emailPreferences,
-      booleanNotation: booleanNotation,
-      displaySettings: displaySettings,
-      verificationDetails: verificationDetails,
-      otherInformation: otherInformation,
-      school: school,
-    });
+    if (recaptchaRef.current) {
```

Comment:
Can't `recaptchaRef` technically be `null` here? You're after all setting it initially as `null`. Shouldn't this be `recaptchaRef?.current` instead? The same goes for the `StudentRegistration` component.

Reply:
The submit button is greyed out until recaptcha is ticked, so handleSubmit won't be called when recaptchaRef is null.